### PR TITLE
feat: support crab-parachain and pangolin parachain

### LINKED ISSUE #204: useControllerAccount

### LINKED ISSUE #205: useStashAccount

### LINKED ISSUE #250: Support Crab parachain and Pangolin parachain

### PR DESCRIPTION
**Pull request by JayJay1024**

### CODE REVIEW

File: src/components/staking/power/Actions.tsx [hunk 1]
```diff
@@ -66,31 +66,35 @@ export function Actions({ eraSelectionIndex, disabled }: ActionsProps) {
...
-        api.tx.staking.withdrawUnbonded?.meta.args.length === 1
-          ? api.tx.staking.withdrawUnbonded(spanCount)
-          : api.tx.staking.withdrawUnbonded(),
-      txSuccessCb: () => {
-        refreshAssets();
-        updateStakingDerive();
-        refreshStakingAccount();
-      },
-    });
+    if (controllerAccount) {
```
Comment (sxlwar) [inline src/components/staking/power/Actions.tsx]: It is better to verify in the execution function, too many changes of the same logic

File: src/components/staking/action/Deposit.tsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ export function Deposit() {
           updateValidators();
           updateStakingDerive();
         }}
-        initialValues={{ stash: stashAccount, promiseMonth: duration, accept: false }}
+        initialValues={{ stash: stashAccount || undefined, promiseMonth: duration, accept: false }}
```
Comment (sxlwar) [inline src/components/staking/action/Deposit.tsx]: Validate in FormModal component, set to undefined if needed

**Review by sxlwar — CHANGES_REQUESTED** [review]

**Review by ZolaxZola — APPROVED** [review]

File: src/components/staking/power/Actions.tsx (hunk 1 above)
Comment (JayJay1024) [reply src/components/staking/power/Actions.tsx]: resolved

File: src/components/staking/action/Deposit.tsx (hunk 2 above)
Comment (JayJay1024) [reply src/components/staking/action/Deposit.tsx]: resolved

**Review by sxlwar — APPROVED** [review]

**Pull request merged**